22-discovery.ks

```diff
@@ -22,6 +22,11 @@ cat > /etc/udev/rules.d/81-nm-prepare.rules <<'UDEV'
 ACTION=="add", SUBSYSTEM=="net", NAME!="lo", RUN+="/usr/bin/systemd-cat -t nm-prepare /usr/bin/nm-prepare %k"
 UDEV
 
+echo " * disable IPv6"
+cat > /etc/sysctl.d/40-ipv6.conf <<EOIP
+net.ipv6.conf.all.disable_ipv6 = 1
+EOIP
```

Comment:
Also I noticed that IPv6 is disabled in nm-configure but not globally for PXE mode.